editor/test/integration/save_data_test.clj

```diff
@@ -470,7 +470,15 @@
 
    'dmRenderDDF.MaterialDesc.Sampler
    {:default
-    {"texture" :unimplemented}}}) ; Default texture resources not supported yet.
+    {"texture" :unimplemented}} ; Default texture resources not supported yet.
+
+   'dmRenderDDF.RenderTargetDesc.DepthStencilAttachment
+   {:default
+    {"format" :unimplemented}} ; Non-default depth/stencil format not supported yet.
+
+   'dmRenderDDF.RenderPrototypeDesc
+   {:default
+    {"materials" :deprecated}}})
```

Comment:
We try to keep this list sorted, so `dmRenderDDF.RenderPrototypeDesc` should really come before `dmRenderDDF.RenderTargetDesc.DepthStencilAttachment`.